common/djangoapps/xblock_django/models.py

```diff
@@ -40,3 +42,42 @@ def disabled_block_types(cls):
             return ()
 
         return config.disabled_blocks.split()
+
+
+class XBlockDeprecatedAdvancedComponentConfig(ConfigurationModel):
```

Comment:
I don't agree with this as they both should be independent entries by merging them into one model would make them coupled. See if I add some entries for `XBlockDisableConfig ` and at this time and then add some entries for `XBlockDeprecatedAdvancedComponentConfig `. Later if I decide to just disable  `XBlockDisableConfig `, can't do it without disabling the `XBlockDeprecatedAdvancedComponentConfig ` as well.